content/applications/finance/accounting/customer_invoices.rst

```diff
@@ -4,225 +4,230 @@
 
-.. image:: customer_invoices/invoice06.png
+Follow-up Reports
+-----------------
 
-Reconcile
----------
+To set the different follow-up levels, go to
+:menuselection:`Accounting --> Configuration --> Follow-up Levels`. All actions triggered after a
+specified number of days can be viewed. To manage the various options, such as the number of days,
+the type of reminder sent, or whether the action is automated, click on the specific line that needs
+to be updated.
 
-Now let's reconcile!
+Then, follow these steps to take the appropriate action:
 
-.. image:: customer_invoices/invoice07.png
+- To access the list of customers requiring action along with the :guilabel:`Total Due` and
+  :guilabel:`Total Overdue` amounts, go to
+  :menuselection:`Accounting --> Customers --> Follow-up Reports`.
+- To get a full overview of the customer's open invoices, click on its line.
+- To open a payment reminder to be sent to the customer, click the :guilabel:`Follow up` button.
+- To exclude any invoices from the follow-up, enable the option :guilabel:`Exclude from Follow-ups`
+  for the selected ones.
 
-You can now go through every transaction and reconcile them or you can mass reconcile with instructions at the bottom.
+.. _accounting/invoices/aging-report:
 
-After reconciling the items in the sheet, the related invoice will now
-display "You have outstanding payments for this customer. You can
-reconcile them to pay this invoice. "
+Aged Receivable
+---------------
 
-.. image:: customer_invoices/invoice08.png
+To review outstanding customer invoices and their related due dates, use the
+:ref:`Aged Receivable <accounting/reporting/aged-receivable>` report. To access it, go to
+:menuselection:`Accounting --> Reporting --> Aged Receivable`.
 
-.. image:: customer_invoices/invoice09.png
+.. _accounting/invoice/reporting:
 
-Apply the payment. Below, you can see that the payment has been added to
-the invoice.
+Reporting
+=========
 
-.. image:: customer_invoices/invoice10.png
+Profit and Loss
+---------------
 
-Payment Followup
-----------------
+The :ref:`Profit and Loss <accounting/reporting/profit-and-loss>` statement shows details of income
+and expenses.
 
-There's a growing trend of customers paying bills later and later.
-Therefore, collectors must make every effort to collect money and
-collect it faster.
+Balance sheet
+-------------
 
-Odoo will help you define your follow-up strategy. To remind customers
-to pay their outstanding invoices, you can define different actions
-depending on how severely overdue the customer is. These actions are
-bundled into follow-up levels that are triggered when the due date of an
-invoice has passed a certain number of days. If there are other overdue
-invoices for the same customer, the actions of the most overdue invoice
-will be executed.
+The :ref:`Balance Sheet <accounting/reporting/balance-sheet>` summarizes the company's liabilities,
+assets, and equity at a specific time.
```

Comment:
```suggestion
The :ref:`Balance Sheet <accounting/reporting/balance-sheet>` summarizes the company's assets,
liabilities, and equity at a specific time.
```
I'm not sure why but it sounds odd to me to have "liabilities" first - maybe because they're often taught with the acronym ALE to remember Assets = Liabilities + Equity, so let's use that order :)